jsonld-contexts/c2jn-compound.jsonld

```diff
@@ -1,11 +1,12 @@
 {
   "@context": [
     "https://easy-global-market.github.io/ngsild-api-data-models/weather/jsonld-contexts/weather.jsonld",
+    "https://easy-global-market.github.io/c2jn-data-models/jsonld-contexts/datacenter-equipments.jsonld",
```

Comment:
create another compound with things specific to industry use-case and the common things (TP, DCAT, authz, core context), to avoid mixing both